docker_mcp/services/host.py

```
@@ -4,6 +4,8 @@
 Business logic for Docker host management operations.
 """
 
+import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)